packages/web/src/hooks/useTabs/useTabs.tsx

```diff
@@ -268,20 +269,21 @@ const TabBar = memo(
           }
 
           return (
-            <Tooltip
-              text={tab.disabledTooltipText || disabledTabTooltipText}
-              placement='bottom'
-              disabled={!tooltipActive}
-              key={i}
-            >
-              {to && pathname ? (
-                <SeoLink {...rootProps} to={`${pathname}/${to}`}>
-                  {tabElement}
-                </SeoLink>
-              ) : (
-                <div {...rootProps}>{tabElement}</div>
-              )}
-            </Tooltip>
+            <ClientOnly key={i}>
```

Comment:
If we want to roll forward with these changes for now, that's probably okay, but i'd like try if we can to keep the tabs, since they have the "seo-link" in them. we want the ssr'd page to have as many quality links as possible. down to chat more about this in general to figure out which links are important to keep around!

Reply:
oh i missed this before. The tooltip was causing an error server-side, but tomorrow ill see if I can conditionally render the links without the tooltip server-side to keep them 👍

Reply:
Nice yeah maybe we can update our tooltips to work in SSR context? Prob just render children or something